### ISSUE TITLE
Files become read-only when latexmk is compiling under Windows

### ISSUE THREAD

**Issue by etnoy** [lervag/vimtex#915]
This is more of a higher-level issue that might not have to do with vimtex, but it heavily interferes with my workflow.

Under windows we all know files are read-only when they are opened in another program. (My least favorite feature, but that's another discussion). This interferes with continuous compilation since latexmk locks my manuscript file(s) and I can't save with :w anymore. I have to use :w! instead which is far from optimal.

For most documents, compilation is fast so this isn't even noticed, however I'm currently writing my PhD thesis and I'm at 200 pages with a myriad of packages, tikz etc. Compilation time is around 30 seconds, so background compilation is great while I type.

Is there an option in vimtex where I can tell latexmk not to lock file files while compiling? Even when using :w! latexmk happily takes the new update and recompiles, so it's not an issue to force save.

Again, maybe this isn't a vimtex problem per se, but I'm sure more people than myself experience this problem.

**Comment by lervag** [comment]
I don't think every application will make a file read-only when opened on windows. SumatraPDF, for instance does not lock the pdf file when it is read. And I did not think latexmk locked the tex file(s) while compiling.

I've searched for more information on this issue, and I can't find anything. However, I can say that this is not something deliberately done by vimtex, and there are no vimtex options available. I don't know how to prevent this behaviour, if even possible (except changing OS).

Sorry!
